procs/connect.sh

```diff
@@ -0,0 +1,5 @@
+export SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+export INPUT_PATH=$1
+
+echo "connecting to community $INPUT_PATH"
+IPFS_PATH=$INPUT_PATH ipfs daemon
```

Comment:
Newline + could we have these be added in apps/communities, and then installed somehow into procs? Just because in general procs should not have anything commited in it (apps.yml is a dumb exception right now until we can generate it at build time)